overview.tt

```diff
@@ -0,0 +1,279 @@
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    [% PROCESS svg path="site-styles/assets/gfx-overview-looking.svg" %]
+    <div>
+      <h2>
+        Either it works or fails always in the same way.
+      </h2>
+      <p>
+        With Nix there are two results: the code either works or fails always
+        in the same way. This gives you the power to reproduce (and avoid) any
+        build related bugs .
+      </p>
+      <ul>
+        <li>reproducibility</li>
+        <li>transparent source binary model</li>
+        <li>nix language</li>
+        <li>continuous integration</li>
+      </ul>
+      <div>
+        <p>
+          Trying to keep system dependencies of development environments in
+          sync with continuous integration (CI) is a disaster waiting to
+          happen. Being able to use the same environment locally or anywhere
+          else is putting a developer back in control.
+        </p>
+        <p>
+          Build instructions are written in Nix language, which allows you to
+          extend or change existing build instruction in any way you wish. This
+          makes Nix a powerful build tool that can be used in both simple and
+          complex projects.
+        </p>
+      </div>
+    </div>
+  </div>
+  <div>
+    [% PROCESS svg path="site-styles/assets/gfx-overview-cloud.svg" %]
+    <div>
+      <h2>
+        From zero to cloud.
+      </h2>
+      <ul>
+        <li>cloud images</li>
+        <li>docker images</li>
+        <li>devops</li>
+      </ul>
+      <p>
+        Use the same tool to build one project or even one hundred projects,
+        configure machines and deploy to the cloud. Thanks to the extensible
+        Nix language, it's easy to compose parts together with little overhead.
+      </p>
+      <div>
+        <p>
+          Use one tool to develop, test and build Amazon EC2, Google Cloud,
+          Azure, Virtualbox, KVM and other images in a declarative way is a
+          DevOps dream.
+        </p>
+        <p>
+          Forget <code>Dockerfile</code> and
+          have a peace of mind with declarative approach Nix gives you. Nix
+          automatically calculates the optimal layer layout and only includes
+          software needed.
+          <strong>Nix and Docker is a match made in heaven.</strong>
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    [% PROCESS svg path="site-styles/assets/gfx-overview-gears.svg" %]
+    <div>
+      <h2>
+        Is the previous version better? Rollback anytime.
+      </h2>
+      <p>
+        Packages are never overwritten, and old versions are always stored.
+        Rollback to any previous version within milliseconds.
+      </p>
+      <ul>
+        <li>generations</li>
+        <li>garbage collection</li>
+        <li>atomic</li>
+        <li>nix store</li>
+      </ul>
+      <div>
+        <p>
+          Since Nix never overwrites files in the Nix store, but just adds new
```

Comment:
The term "Nix store" hasn't been introduced. Maybe it can be omitted ("Nix never overwrites files in place, but just adds...").